### PR TITLE
Add layout

### PR DESCRIPTION
**Pull request by zubrik1**
I will change in the process of work.

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+}
+.forecastDetails .toggle{
+  background-color: #111111;
+  color: #858484;
+  border: 2px groove #111111;
+  -webkit-box-shadow: #000 1px 1px 10px inset;
+  box-shadow: #000 1px 1px 10px inset;
+}
+#forecastPeriod{
+  font-family: "Kalam", cursive;
```
Comment (viktor-yakubiv) [inline css/main.css]: It's better to use more strong font, probably `sans-serif`. Cursive looks funny but it makes batman look not so powerful as he is.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Weather app</title>
+    <link rel="stylesheet" href="css/reset.css">
```
Comment (viktor-yakubiv) [inline index.html]: Remove reset completely. We __do not__ need it. Everything we use from this whole file is `body { margin: 0; }`

File: css/main.css [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+  margin: 0 auto;
+  text-align: center;
+  border: 2px groove #111111;
+  -webkit-animation: type 6s steps(50, end);
+  animation: type 6s steps(50, end);
+  outline: none;
+  -webkit-box-shadow: #000 1px 1px 10px inset;
+  box-shadow: #000 1px 1px 10px inset;
+}
+@keyframes type {
```
Comment (viktor-yakubiv) [inline css/main.css]: Animation does not look so good. I'd prefer to remove it.

Anyway, usage of `width`, `height`, `margin`, `padding` for animation harms performance. It's better to use `transform`. Use it if you really want to.

File: css/main.css [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+  padding: 0;
+  margin: 0;
+}
+.mainSearch {
+  height: 70vh;
+}
+.mainSearch .logo {
+  height: 60%;
+  width: 100%;
+  background-image: url(../img/main.png);
```
Comment (viktor-yakubiv) [inline css/main.css]: Prefer shorthands. Only `background`, for this and 3 following rows; that's enough.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+            placeholder="Enter the city that needs a hero... or forecast"
+            onkeypress="onKeyPress(event)"/>
+
+        <div class="recentCities">
+            <div class="recentItem">Kiev</div>
+            <div class="recentItem">Odessa</div>
+        </div>
+
+        <div class="forecastDetails">
+            <button onclick="toggleScale('metric')" class="toggle">°C</button>
```
Comment (viktor-yakubiv) [inline index.html]: This if not `button`. It's `input[type=radio]`. Do it in that way.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+            <button onclick="toggleScale('imperial')">°F</button>            
+           
+            <select name="forecastPeriod" id="forecastPeriod" onchange="togglePeriod()">
```
Comment (viktor-yakubiv) [inline index.html]: I guess we do not need period but leave it for this moment.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Weather app</title>
+    <link rel="stylesheet" href="css/reset.css">
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/main.css">
+    <link href="https://fonts.googleapis.com/css?family=Kalam" rel="stylesheet">    
+    <script defer src="https://use.fontawesome.com/releases/v5.0.6/js/all.js"></script>
```
Comment (viktor-yakubiv) [inline index.html]: Do we use FontAwesome somewhere?

File: index.html [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+</head>
+
+<body>
+    <section class="mainSearch">
+        <div class="logo"></div>
```
Comment (viktor-yakubiv) [inline index.html]: Why is it empty?

Logo should be an image with alternative test (attribute) what describes it and service in all if possible.

File: index.html (hunk 7 above)
Comment (zubrik1) [reply index.html]: Not now, but in future)

Comment (viktor-yakubiv) [reply index.html]: We can add in the future. Product must run without anything unnecessary each time of life.

**Comment by viktor-yakubiv** [comment]
Resolves #7

**Pull request merged**